packages/core/core/test/TargetResolver.test.js

```diff
@@ -8,6 +8,9 @@ import {DEFAULT_OPTIONS} from './utils';
 
 import TargetResolver from '../src/TargetResolver';
 
+//$FlowFixMe
+const rejects = assert.rejects;
```

Comment:
why not `assert.rejects` below?

Reply:
I wanted to avoid a `//$FlowFixMe` comment on every assert, and two `it` test cases use the function. Should I completely remove this declaration or make two local variables in `it`?